tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import collections
+import os
+
+from lsst.meas.astrom import LoadAstrometryNetObjectsTask, LoadAstrometryNetObjectsConfig
+from lsst.afw import geom, table
+
+from lsst.jointcal import jointcal
+
+arcsec_per_radian = 206265.
+
+
+class JointcalTestBase(object):
+    """
+    Base class for jointcal tests, to genericize some test running and setup.
+
+    Derive from this first, then from TestCase.
```

Comment:
I think you need to say something about entry points, because having had a quick look over this class, I don't see what the user is supposed to call.  There's `setUp` and `tearDown`, but those are for `TestCase`, not the real user.  Then there's `computeRms` which is nice, but I get the feeling that this class is supposed to do more.  I would have thought `_testJointCalTask` and `_testJointCalTask_run` were the entry points, but they're 'declared' as private with the leading underscore (which I interpret as "no-one outside this class should ever have to call this").